src/library.cpp

```diff
@@ -15,7 +17,7 @@ QString Library::openBook(const QString &zimPath)
         if(QString::fromStdString(it->second->getZimFilePath()) == zimPath)
             return it->first;
     }
-    qInfo() << "Opening" << zimPath;
+    qInfo() << QObject::tr("Opening") << zimPath;
```

Comment:
I'm not sure we need to translate log messages.

Reply:
effectively.